applications/LinearSolversApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -0,0 +1,56 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define_python.h"
+
+// Application includes
+#include "custom_python/add_custom_utilities_to_python.h"
+#if defined(USE_EIGEN_MKL)
+#include "custom_utilities/mkl_utilities.h"
+#endif
+
+namespace Kratos::Python {
+
+void AddCustomUtilitiesToPython(pybind11::module& m)
+{
+    namespace py = pybind11;
+
+#if defined(USE_EIGEN_MKL)
```

Comment:
Not sure if its better to disable completely 

Maybe would be easier to issue warnings in the actual code if its called but mkl is not compiled

Just a thought